backend/experiments/cli/LibCli.fs

```diff
@@ -55,8 +55,23 @@ let fns : List<BuiltInFn> =
       previewable = Impure
       deprecated = NotDeprecated }
 
-
-
-
-
-    ]
+    { name = fn "File" "write" 0
```

Comment:
Would it be better to move the function to a separate file in order to ensure that the two experiments remain independent of each other?